models/Groups.js

```diff
@@ -0,0 +1,37 @@
+//literal expression to indicate code should be executed in 'strict mode' which hels you write cleaner code
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+    const Groups = sequelize.define('Groups', {
+
+        //auto generate UUID format - can be UUIDV1 or UUIDV4
+        uuid: {
+            type: DataTypes.UUID,
+            primaryKey: true,
```

Comment:
Let's remove `primaryKey` here

Reply:
- Going to leave primaryKey which replaces auto id